XML/TestCases/FunctionalTests.xml

```diff
@@ -4,6 +4,9 @@
         <testScript>VERIFY-DEPLOYMENT-PROVISION.ps1</testScript>
         <files></files>
         <setupType>OneVM</setupType>
+        <TestParameters>
+            <param>vCpu=VCPU</param>
```

Comment:
add a comment here that This represents the custom expected number of cores inside the VM.

Reply:
done